module-telemetry/src/main/java/com/mapbox/maps/module/telemetry/MapBaseEvent.kt

```diff
@@ -1,16 +1,13 @@
 package com.mapbox.maps.module.telemetry
 
 import android.annotation.SuppressLint
-import android.os.Parcel
 import com.google.gson.annotations.SerializedName
-import com.mapbox.android.telemetry.Event
 
 /**
  * Base event class for telemetry events.
  */
 @SuppressLint("ParcelCreator")
-internal abstract class MapBaseEvent(phoneState: PhoneState) :
-  Event() {
+internal abstract class MapBaseEvent(phoneState: PhoneState) {
```

Comment:
Need to double check the Event class in MME and see if we miss anything.

Reply:
refs https://github.com/mapbox/mapbox-events-android/blob/d73ca293b194376516997b73c5139b28430be671/libtelemetry/src/main/java/com/mapbox/android/telemetry/Event.java

Reply:
Event type is determined from `event`  key in json, for MapLoad value should be `map.load`. Event should respect the [schema](https://github.com/mapbox/event-schema/blob/master/lib/base-schemas/map.load.js). Which means that valid event should have `type = map.load`, and also valid `userId`  and `created`  fields and may have some of optional fields. We do not serialize class type, I see no issues here.

valid `map.load` event example:
```
[{"wifi":false,"cellularNetworkType":"Unknown","orientation":"Portrait","userId":"d1cec2d6-5966-4608-9e85-59ac51178de0","sdkVersion":"10.8.0-SNAPSHOT","created":"2022-09-13T14:47:47.944+0300","sdkIdentifier":"mapbox-maps-android","event":"map.load","operatingSystem":"Android - 13","resolution":2.75,"carrier":"T-Mobile","model":"sdk_gphone64_arm64","pluggedIn":false,"accessibilityFontScale":1.0,"batteryLevel":100}]
```

Reply:
checked the event body we sent and I see no issue either, thanks for the confirmation!